### ISSUE TITLE
option pour non prise en compte des prénoms lors de la création de fichiers tabulés

### ISSUE THREAD

**Issue by msaby** [Transition-bibliographique/bibliostratus#65]
parfois une tentative d'alignement peut échouer car le catalogue cible (je pense au Sudoc) contient uniquement des initiales dans les zones 70X $b alors que le catalogue source contient des prénoms complets.
une option permettant de ne pas prendre en compte cette sous-zone lors de la création de fichiers tabulés à partir de notices pourrait être intéressante

**Comment by Lully** [comment]
oui, c'est techniquement possible, mais je ne suis pas sûr que ce soit utile à ce moment-là du programme
En effet, ça reste toujours plus intéressant de commencer par rechercher avec le prénom complet. Ce n'est qu'en cas d'échec sur le prénom complet qu'on peut avoir envie de faire une recherche sur l'initiale (faire un alignement d'emblée sur l'initiale me semble risqué). 
Donc en supposant qu'un utilisateur ait lancé un premier alignement sur 100 notices, et que dans 25 cas ça n'ait rien donné, il voudra peut-être relancer sur l'initiale du prénom uniquement. Mais dans ce cas, aucun intérêt de repartir du fichier Marc pour reconvertir les 100 notices en 100 lignes : autant retravailler directement sur le tableau de résultats, en filtrant sur les 0 alignement trouvé, et là nettoyer les prénoms, non ?
(je soupçonne que tu vas me dire qu'une fois le nom et prénom mis dans la même case, ça ne va pas être simple de savoir lequel des deux doit être tronqué)
